lib/routes/S3_store_router.js

```
@@ -1287,5 +1287,6 @@ class S3RequestLogger extends NodeHttpHandler {
 }
 
 function stripQuotes (ETag) {
-  return ETag.replace(/^"|^W\/"|"$/g, '');
+  if ('string' !== typeof ETag) { return ETag; }
```

ESLint

[error] Expected literal to be on the right side of !==.

(yoda)